### ISSUE TITLE
Fluentbit can not send datas to outputs, store them to filesystem

### ISSUE THREAD

**Issue by merveyilmaz-netrd** [fluent/fluent-bit#4360]
## Bug Report

**Describe the bug**
I have a bash script that generates 500 mb data per minute. I tail these logs and send them to different outputs. I work with Elastic, Splunk and Kafka. I am using filesystem buffering with mem_buf_limit option and set it to 500MB. I saw that Fluentbit stores the data quickly in filesystem.  But it cannot send the data to the outputs as fast as it stores them in the filesystem. For example, I run bash script for 5 minutes and it generates a log file around 2.5GB. And i saw that fleuntbit almost store all datas in the filesystem(2.3 gb in filesystem). Why fluentbit can not send datas quickly?

**To Reproduce**
- Rubular link if applicable:
- Example log message if applicable:
```
{"log":"YOUR LOG MESSAGE HERE","stream":"stdout","time":"2018-06-11T14:37:30.681701731Z"}
```
- Steps to reproduce the problem:
setting mem_buf_limit to 1GB which does not work.

**Expected behavior**

**Screenshots**

**Your Environment**

* Version used: 1.8.8
* Configuration:

* Environment name and version (e.g. Kubernetes? What version?):
* Server type and version: Centos7.9
* Operating System and version:
* Filters and plugins: no filter. tail input plugin. Elastic splunk and kafka output plugin.

**Additional context**

**Comment by nokute78** [comment]
It may be same issue https://github.com/fluent/fluent-bit/issues/4221

**Comment by edsiper** [comment]
Fixed in #4411

**Comment by merveyilmaz-netrd** [comment]
> Fixed in #4411

I am installing fluentbit with rpm. Does Latest release (1.8.10) contain this fix?
